cdk_framework/EKS/lib/stacks/fargate-cluster-stack.ts

```diff
@@ -0,0 +1,38 @@
+import { Stack, StackProps, aws_eks as eks} from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import { Vpc } from 'aws-cdk-lib/aws-ec2';
+import { KubernetesVersion} from 'aws-cdk-lib/aws-eks';
+import { Role } from 'aws-cdk-lib/aws-iam';
+
+
+export class FargateStack extends Stack {
```

Comment:
Does this stack require any sort of IAM policies like the ec2 worker roles?

Reply:
There is a default role provided assumed by anyone with permissions in the same account